exotica_python/src/pyexotica/interactive_rho_tuning.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import print_function
+import os
+from time import time
+import Tkinter as tk
+import rospkg as rp
+
+__all__=['Tuning']
+
+class Tuning(object):
```

Comment:
Can you please select a more descriptive name for this class, e.g., `UnconstrainedEndPoseProblemInteractiveCostTuning` or `InteractiveCostTuning`. As far as I can tell it now only supports costs, not constraints - and only for end-pose problems.